extensions/skins/conversation_v1/player.html

```diff
@@ -30,6 +30,35 @@
                    ng-class="{'conversation-skin-progress-dots-iframed': isIframed, 'conversation-skin-progress-dots-not-iframed': !isIframed}">
     </progress-dots>
 
+    <div ng-if="helpHtml" class="conversation-skin-help-card-container">
```

Comment:
Why is a container needed here? Seems to me the ng-if can be directly on the .conversation-skin-help-card, and all of the positioning CSS can apply directly to the card without having a container.